tox.ini

```diff
@@ -10,7 +10,7 @@ whitelist_externals = sh
 
 [testenv:cov]
 basepython = python2.7
-commands = pytest --cov-report=html --cov=alt_src {posargs}
+commands = pytest --cov-report=xml --cov=alt_src {posargs}
```

Comment:
I think it'd be good to enable both the html and xml outputs.  The html output is user-readable and that's why it was enabled in the first place.  Now the setup is less friendly to developers who want to check the coverage of their local changes.

I'll still approve anyway since this is fixing the problem it's intended to fix, and I won't repeat this comment on all the similar PRs

Reply:
Updated to include both xml and html coverage reports.